RIGS/templates/RIGS/event_detail_buttons.html

```diff
@@ -0,0 +1,33 @@
+<div class="btn-group btn-page">
+    <a href="{% url 'event_update' event.pk %}" class="btn btn-default"><span
+            class="glyphicon glyphicon-edit"></span> <span
+            class="hidden-xs">Edit</span></a>
+    {% if event.is_rig %}
+        <a href="{% url 'event_print' event.pk %}" target="_blank" class="btn btn-default"><span
+                class="glyphicon glyphicon-print"></span> <span
+                class="hidden-xs">Print</span></a>
+    {% endif %}
+    <a href="{% url 'event_duplicate' event.pk %}" class="btn btn-default" title="Duplicate Rig"><span
+            class="glyphicon glyphicon-duplicate"></span> <span
+            class="hidden-xs">Duplicate</span></a>
+    {% if event.is_rig %}
+        <a class="btn btn-default item-add modal-href event-authorise-request"
+                href="{% url 'event_authorise_request' object.pk %}">
+            <span class="glyphicon glyphicon-send"></span>
+            Authorisation Request
```

Comment:
I'm not against tracking ti for DB storage, wrong phrasing, more complexity of models, which would rapidly grow to have many to many relationships. As I said, current system is stateless, using HMAC. I don't think it would be an issue, but if it is then we can look into it again.